operations/validate.go

```diff
@@ -50,20 +50,12 @@ func Validate() cli.Command {
 			if err != nil {
 				return err
 			}
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
 
 			conf, err := NewClientSettings(confPath)
 			if err != nil {
 				return errors.Wrap(err, "loading configuration")
 			}
 
-			client, err := conf.setupRestCommunicator(ctx, !quiet)
```

Comment:
code clean up- wasn't used anywhere and same with the context once you remove this.

Reply:
Probably someone once having the small hope that we would one day get rid of the legacy REST API and use REST v2 🤷